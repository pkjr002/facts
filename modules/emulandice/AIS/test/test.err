trying URL 'https://cloud.r-project.org/src/contrib/devtools_2.4.5.tar.gz'
Content type 'application/x-gzip' length 374718 bytes (365 KB)
==================================================
downloaded 365 KB

* installing *source* package ‘devtools’ ...
** package ‘devtools’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (devtools)

The downloaded source packages are in
	‘/tmp/RtmpTmLzIe/downloaded_packages’
Skipping 1 packages not available: dummies
Installing 1 packages: stringi
trying URL 'https://cloud.r-project.org/src/contrib/stringi_1.7.8.tar.gz'
Content type 'application/x-gzip' length 8032287 bytes (7.7 MB)
==================================================
downloaded 7.7 MB

* installing *source* package ‘stringi’ ...
** package ‘stringi’ successfully unpacked and MD5 sums checked
** using staged installation
ERROR: configuration failed for package ‘stringi’
* removing ‘/cache/home/rk509/.conda/envs/conda-entk/lib/R/library/stringi’
* restoring previous ‘/cache/home/rk509/.conda/envs/conda-entk/lib/R/library/stringi’

The downloaded source packages are in
	‘/tmp/RtmpdWQ1qP/downloaded_packages’
Warning message:
In i.p(...) : installation of package ‘stringi’ had non-zero exit status
* installing to library ‘/cache/home/rk509/.conda/envs/conda-entk/lib/R/library’
* installing *source* package ‘emulandice20221024160631’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (emulandice20221024160631)
Loading required package: tidyverse
── Attaching packages ─────────────────────────────────────── tidyverse 1.3.2 ──
✔ ggplot2 3.3.6      ✔ purrr   0.3.5 
✔ tibble  3.1.8      ✔ dplyr   1.0.10
✔ tidyr   1.2.1      ✔ stringr 1.4.1 
✔ readr   2.1.3      ✔ forcats 0.5.2 
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
Warning messages:
1: package ‘tibble’ was built under R version 4.2.1 
2: package ‘purrr’ was built under R version 4.2.1 
3: package ‘stringr’ was built under R version 4.2.1 
Warning message:
In for (i in seq_len(n)) { :
  closing unused connection 3 (/scratch/rk509/test.1666641988/emulandice-master/../results/output.txt)
Removing from library ‘/cache/home/rk509/.conda/envs/conda-entk/lib/R/library’
mv: cannot stat ‘*.nc’: No such file or directory
