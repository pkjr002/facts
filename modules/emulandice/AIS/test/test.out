Initiating test in /scratch/rk509/test.1668006033...

Extracting data files...

Executing workflow...
source emulandice_environment.sh --Rscript
[1] "____________________________________________________"
[1] "Welcome to emulandice"
[1]  
[1] "Can ignore the warnings from optimize"
[1] "____________________________________________________"
python emulandice_preprocess.py --pipeline_id AIS-emulandice-AIS-ssp585 --baseyear 2005 --input_data_file test/input/fair-temperature-fair-ssp585_gsat.nc
python emulandice_fit_AIS.py --pipeline_id AIS-emulandice-AIS-ssp585
python emulandice_project_AIS.py --pipeline_id AIS-emulandice-AIS-ssp585
Rscript -e source('packrat/init.R') -e library(emulandice) -e emulandice::main('decades',dataset='FACTS_CLIMATE_FORCING.csv',N_FACTS=500,ice_sources=c('AIS'))
python emulandice_postprocess_AIS.py --pipeline_id AIS-emulandice-AIS-ssp585 --locationfile /home/rk509/facts/modules/emulandice/AIS/test/../../../../scripts/moduletest/location_list.lst --chunksize 600

Collecting output files...
AIS-emulandice-AIS-ssp585_EAIS_globalsl.nc
AIS-emulandice-AIS-ssp585_EAIS_localsl.nc
AIS-emulandice-AIS-ssp585_globalsl.nc
AIS-emulandice-AIS-ssp585_localsl.nc
AIS-emulandice-AIS-ssp585_PEN_globalsl.nc
AIS-emulandice-AIS-ssp585_WAIS_globalsl.nc
AIS-emulandice-AIS-ssp585_WAIS_localsl.nc

Cleaning up...
